### DISCUSSION TITLE
Confirmed - Works on the Korg Triton series

### DISCUSSION THREAD

**Discussion by manpaz** [erichelgeson/BlueSCSI-v1#169] [Show and tell]
Hello,

I'd like to confirm that the BlueSCSI works on the Korg Triton series, although I only have access to the Korg Triton-Rack, the board to add the SCSI capabilities seems to be pretty much the same for all the series, the EXB-SCSI (KLM-2087) based on Fujitsu MB86604L.

Tested with a BlueSCSI 1.1a DB25

Based on the instructions of the Parameter Guide, I created 4 drives of 4GB each, on the LUN0 and assigned them to the SCSI IDs 1, 2, 3, and 4 to avoid conflicts with the ID 0 which is usually reserved for communications with external computers, although, I couldn't find anything on the documentation about using the SCSI port for communications or anything else other than media/disks attachment, thus there is no mention as well of any ID reserved for the Triton's SCSI port, however, there is the assumption that the Triton uses the ID 7 because there is suggestion to use the IDs 0 to 6 for all the external devices.

Technically, one can go with up to 8 IDs, minus the ID 7 which is taken by the Triton, then, there are 7 IDs still available, however, for the purpose of testing, and because I'm using a 16GB SD, I only created 4 drives.

The procedure used is as follows:

1. On Linux, execute the following commands to produce the needed volumes:
```
dd if=/dev/zero of=HD10_512.hda bs=4M count=1024
dd if=/dev/zero of=HD20_512.hda bs=4M count=1024
dd if=/dev/zero of=HD30_512.hda bs=4M count=1024
dd if=/dev/zero of=HD40_512.hda bs=4M count=1024
```
2. Once the volumes are done, format the SD card as ExFAT and then copy over the images into the SD card.
3. Umount the SD card
4. Plug-in the SD card to the BlueSCSI, hook up the BlueSCSI to the SCSI port on the Triton and then power up the Triton.
5. Once the unit is on, press the "DISK" button
6. Press F4 ("Inform") and scroll the "VALUE" knob until "SCI ID: 1" is shown on the second row on the left at the screen
7. Press F3 ("Utility") and the press F8 ("UTILITY")
8. Use the arrows to move the cursor until "Format" is highlighted, and then press F8 ("OK")
9. Press F5 ("Name") if you want to set a volume label, then press F8 ("OK") if you want to format the volume, and confirm when prompted
10. Repeat the steps 6 to 9 for SCSI IDs 2,3, and 4
11. Turn off the Triton, detach the BlueSCSI from the SCSI port on the Triton, take the SD card and put it back on the computer
12. Mount the SD Card on the OS
13. Get the partition offset and size
```
fdisk -l /run/media/$(whoami)/BlueSCSI/HD10_512.hda

Device                                       Boot Start     End Sectors Size Id Type
/run/media/<username>/BlueSCSI/HD50_512.hda1 *       32 8388607 8388576   4G  b W95 FAT16
``` 
14. Mount the first volume (Assuming that you labeled the SD Card BlueSCSI and using the "Start" and "Sectors" values from above)
```
sudo mount -o loop,offset=$((512*32)),sizelimit=$((512*8388576)) /run/media/$(whoami)/BlueSCSI/HD10_512.hda /mnt/
```
15. Mount the first volume (Assuming that you labeled the SD Card BlueSCSI)
```
sudo mount -o loop,offset=16384,sizelimit=4294934528 /run/media/$(whoami)/BlueSCSI/HD10_512.hda /mnt/
```
16. Copy all your files into the /mnt directory
```
cp -r /path/to/my/samples /mnt/
```
17. Umount the volume
```
sudo umount /mnt
```
18. Umount the SD card
19. Plug-in the SD card to the BlueSCSI, hook up the BlueSCSI to SCSI port on the Triton and then power up the Triton
20. Turn the Triton on again, press "DISK" button
21. Press F4 ("Inform") and scroll the "VALUE" knob until "SCI ID: 1" is shown on the second row on the left at the screen
22. Press F1("Load") and you should be able to see your files and navigate through the volume on the SCSI 1
23. Select your PCM and SNG as usual and load them on "I-E"
24. Play the sounds on the Internal Bank E and they should work fine, unless the sampling RAM is defective.

Hope that helps others looking for a solution for their Korg Triton.

Thank you!

**Comment by AJRubenstein** [comment]
Hi @manpaz - I would like to know, did you power the BlueSCSI separately or off the SCSI port? I'm able to get it to work with external power, but it won't work without it. But strangely, the power lights do work when plugged into the port without power.

**Reply by manpaz** [reply]
Hi @AJRubenstein , I didn't need any external power, just plugged in the BlueSCSI to the Triton-Rack and it was able to see and operate the SCSI HDs created in the SD card.

**Reply by AJRubenstein** [reply]
AH! I missed the part where it was a Triton Rack, I'm using a Triton classic.
